beacon_node/beacon_chain/src/beacon_fork_choice_store.rs

```diff
@@ -75,13 +75,11 @@ pub(crate) struct CacheItem {
 pub(crate) type CacheItem = CacheItemV8;
 
 #[superstruct(
-    variants(V1, V8),
+    variants(V8),
```

Comment:
Do we need to keep the V8 variant here? I suppose it makes it easier for when it is updated in the future since the boilerplate is already there?
Just seems strange to have a V8 variant when there is no longer a V8 schema.

Reply:
Yeah I was super tempted to delete this as well, but figured it might come in handy for future migrations

The naming thing does turn out a bit strange, because the names are the _first_ version in which the thing is introduced for the reasons discussed here: https://github.com/sigp/lighthouse/blob/stable/beacon_node/beacon_chain/src/schema_change/README.md#naming-conventions